src/Media.php

```diff
@@ -62,7 +62,9 @@ public static function boot()
      */
     public function models($class)
     {
-        return $this->morphedByMany($class, 'mediable')->withPivot('tag', 'order');
+        $table = config('mediable.mediables_table') ?: 'mediables';
+
+        return $this->morphedByMany($class, 'mediable', $table)->withPivot('tag', 'order');
```

Comment:
styling, also `config()` accepts a default value, no need for ternary expression.

```php
return $this->morphedByMany(
    $class, 
    'mediable', 
    config('mediable.mediables_table', 'mediables')
)->withPivot('tag', 'order');

Reply:
👍